.env

```diff
@@ -10,7 +10,7 @@ REACT_APP_VX_USE_MOCK_PRINTER=FALSE
 REACT_APP_VX_USE_MOCK_PAPER_HANDLER=FALSE
 REACT_APP_VX_USE_CUSTOM_SCANNER=FALSE
 REACT_APP_VX_SCAN_USE_FUJITSU_PRINTER=FALSE
-REACT_APP_VX_SKIP_CVR_ELECTION_HASH_CHECK=FALSE
+REACT_APP_VX_SKIP_CVR_BALLOT_HASH_CHECK=FALSE
```

Comment:
maybe post in slack so people are aware of this one just in case they use it in their .env.local